### PR TITLE
Removes spawn()s in securitrons

### PR DESCRIPTION
**Pull request by ghost**
:cl: More Robust Than You
code: Removed some spawn()s from secbot code
/:cl:

### CODE REVIEW

**Comment by JJRcop** [comment]
dropping rhymes

File: code/modules/mob/living/simple_animal/bot/secbot.dm [hunk 1]
```diff
@@ -225,21 +224,25 @@ Auto Patrol: []"},
...
-		if( !Adjacent(C) || !isturf(C.loc) ) //if he's in a closet or not adjacent, we cancel cuffing.
-			return
-		if(!C.handcuffed)
-			C.handcuffed = new /obj/item/restraints/handcuffs/cable/zipties/used(C)
-			C.update_handcuffed()
-			playsound(loc, pick('sound/voice/bgod.ogg', 'sound/voice/biamthelaw.ogg', 'sound/voice/bsecureday.ogg', 'sound/voice/bradio.ogg', 'sound/voice/binsult.ogg', 'sound/voice/bcreep.ogg'), 50, 0)
-			back_to_idle()
+	addtimer(CALLBACK(src, .proc/cuff_cancel, C), 60)
+
+/mob/living/simple_animal/bot/secbot/proc/cuff_cancel(mob/living/carbon/C)
```
Comment (kevinz000) [inline code/modules/mob/living/simple_animal/bot/secbot.dm]: should be called attempt_handcuff instead.

File: code/modules/mob/living/simple_animal/bot/secbot.dm [hunk 2]
```diff
@@ -225,21 +224,25 @@ Auto Patrol: []"},
...
+	if( !Adjacent(C) || !isturf(C.loc) ) //if he's in a closet or not adjacent, we cancel cuffing.
+		return
+	if(!C.handcuffed)
+		C.handcuffed = new /obj/item/restraints/handcuffs/cable/zipties/used(C)
+		C.update_handcuffed()
+		playsound(loc, pick('sound/voice/bgod.ogg', 'sound/voice/biamthelaw.ogg', 'sound/voice/bsecureday.ogg', 'sound/voice/bradio.ogg', 'sound/voice/binsult.ogg', 'sound/voice/bcreep.ogg'), 50, 0)
```
Comment (Cyberboss) [inline code/modules/mob/living/simple_animal/bot/secbot.dm]: loc -> src

use get_sfx for the sound picking

**Review by Cyberboss — CHANGES_REQUESTED** [review]
What kev said

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**